sympy/stats/crv_types.py

```diff
@@ -1943,6 +1928,18 @@ def Gumbel(name, beta, mu, minimum=False):
     >>> cdf(X)(x)
     exp(-exp(-(-mu + x)/beta))
 
+    Parameters
+    ==========
+
+    mu : Real number, 'mu' is a location
+    beta : Real number, 'beta > 0' is a scale
```

Comment:
Should these use backticks?

Reply:
Changed to single backticks. Also fixed for ExponentialPower(), Gompertz(), and ShiftedGompertz().

But would it be better to use dollar signs? See https://github.com/sympy/sympy/pull/17605 and https://github.com/sympy/sympy/issues/13519#issuecomment-530151357

There is also an argument for using double backticks to render them as `code`, as implied by the [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections).